src/datasets/model.rs

```diff
@@ -320,6 +320,8 @@ pub struct Query {
     pub apl: String,
     pub start_time: Option<DateTime<Utc>>,
     pub end_time: Option<DateTime<Utc>>,
+    pub cursor: Option<String>,
+    pub include_cursor: Option<bool>,
```

Comment:
Can we make this a non-option?

Reply:
Fixed in https://github.com/axiomhq/axiom-rs/pull/44/commits/ae957d05e8b4de06920875ca85746ab7b6336391